spec/features/feedback_spec.rb

```diff
@@ -80,6 +80,8 @@
       end
 
       background do
+        application.authority.update_attribute(:write_to_councillors_enabled, true)
```

Comment:
@equivalentideas I'm pretty sure I told you about this method so it's my fault :smile: but a more concise version of this is:

```ruby
application.authority.update(write_to_councillors_enabled: true)
```

Unless you're capturing the return value and doing something with it you should also use the bang method so it raises and error if there's a problem (fail fast):

```ruby
application.authority.update!(write_to_councillors_enabled: true)
```